draft-ietf-privacypass-architecture.md

```diff
@@ -822,93 +765,16 @@ Firefox). As a result, log_2(U/A) is approximately 6 and so the maximum
 value of I should be 3.
 
 If the value of U is much bigger (e.g. 5 million) then this would permit
-I = (log_2(5000000/5000)-1)/2 ~= 4 servers.
+I = (log_2(5000000/5000)-1)/2 ~= 4 Issuers.
 
-## Allowing more servers
+## Allowing more Issuers
 
 Using the recommendations in {{more-servers}}, it is possible to
```

Comment:
I do like the approach of having clients limit the number of non-interactive cross-origin tokens they're willing to maintain from different Issuers.